.infra/Pulumi.prod.yaml

```diff
@@ -106,7 +106,7 @@ config:
       secure: AAABAI/bDI+xz5PMlMZseI2kW5oanHk3z1KcAOZJb+/G8arhgSP1EMdqdOTk0kku1t6KTWtl9+oSSDKWYNH1CG5D+Ac1ZPFAWc/ldmEb/iU=
     onesignalAppId:
       secure: AAABALTizkxZkCfsq6/fx2v7nXxsVP7psWBfSUct0zeyrX0ABxPpD30zIsyx5r5wqt1UJIIXDIVOnhM7W3vzrzaatrE=
-    otelEnabled: false
+    otelEnabled: true
```

Comment:
What was the issue again when we disabled?

Reply:
I think it was user search going haywire, causing lots of logs because it never really finished a span

Reply:
It might still occur, haven't had a chance to check it locally. Enabling it so @vdubc can get traces from API

Reply:
Ignore the test fail, looking at it

Reply:
aight 👍